static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic.html',
```

Comment:
Out of interest, what is a bigger bottleneck, the combined size of the files or the number of them?